docs/data/material/getting-started/templates/checkout/Checkout.tsx

```diff
@@ -55,68 +117,297 @@ export default function Checkout() {
+          >
+            <Box
+              sx={{
+                display: { xs: 'flex', md: 'none' },
+                flexDirection: 'row',
+                width: '100%',
+                justifyContent: 'space-between',
+              }}
+            >
+              <Button
+                startIcon={<ArrowBackRoundedIcon />}
+                component="a"
+                href="/material-ui/getting-started/templates/landing-page/"
+                sx={{ alignSelf: 'start' }}
+              >
+                Back to
+                <img
+                  src={
+                    'https://assets-global.website-files.com/61ed56ae9da9fd7e0ef0a967/61f12e6faf73568658154dae_SitemarkDefault.svg'
+                  }
+                  style={logoStyle}
+                  alt="logo of sitemark"
+                />
+              </Button>
+              <ToggleColorMode
+                mode={mode}
+                toggleColorMode={toggleColorMode}
+                showCustomTheme={showCustomTheme}
+              />
+            </Box>
+            <Box
+              sx={{
+                display: { xs: 'none', md: 'flex' },
+                flexDirection: 'column',
+                justifyContent: 'space-between',
+                alignItems: 'flex-end',
+                flexGrow: 1,
+                height: 150,
+              }}
+            >
+              <ToggleColorMode
+                mode={mode}
+                toggleColorMode={toggleColorMode}
+                showCustomTheme={showCustomTheme}
+              />
+              <Stepper
+                id="desktop-stepper"
+                activeStep={activeStep}
+                sx={{
+                  width: '100%',
+                  height: '40px',
+                }}
+              >
+                {steps.map((label) => (
+                  <Step
+                    sx={{
+                      ':first-child': { pl: 0 },
+                      ':last-child': { pr: 0 },
+                    }}
+                    key={label}
+                  >
+                    <StepLabel>{label}</StepLabel>
+                  </Step>
+                ))}
+              </Stepper>
+            </Box>
+          </Box>
+          <Card
+            sx={{
+              display: { xs: 'flex', md: 'none' },
+              width: '100%',
+            }}
+          >
+            <CardContent
+              sx={{
+                display: 'flex',
+                width: '100%',
+                alignItems: 'center',
+                ':last-child': { pb: '16px' },
+              }}
+            >
+              <Box sx={{ flexGrow: 1 }}>
+                <Typography variant="subtitle2" gutterBottom>
+                  Selected products
+                </Typography>
+                <Typography variant="body1">
+                  <Typography variant="body1">
+                    {activeStep === 2 ? '144.97' : '$134.98'}
+                  </Typography>
+                </Typography>
```

Comment:
```suggestion
                <Typography variant="body1">
                    {activeStep === 2 ? '$144.97' : '$134.98'}
                </Typography>
```

This is the hydration error you were seeing, a <p> nested on another